### ISSUE TITLE
Create post - button "Choose file" should appear only after the click to the link "Add file"

### ISSUE THREAD

**Issue by teinett** [pepyatka/pepyatka-html#181]
By defauld there should be a link "Add file" in the block of creating post.

Now I see the button "Choose file". 
This button should appear only after the click to the link "Add file".

Screenshot: https://yadi.sk/i/5xZyuhNQg4Nfx

**Comment by smallq-git** [comment]
This is how it was in Frf:
![image](https://cloud.githubusercontent.com/assets/7983303/7211523/361564ac-e564-11e4-8ab0-0b7efcf73c7c.png)

**Comment by epicmonkey** [comment]
test

**Comment by smallq-git** [comment]
Apparently, there was no need for additional actions after clicking Photos or Files link - new issue is opened for that (https://github.com/pepyatka/pepyatka-html/issues/418). This particular issue is fixed:
![image](https://cloud.githubusercontent.com/assets/7983303/7870861/a97f501a-0594-11e5-8216-c440dc942b4a.png)
